frontend/src/views/reports/SurveyView.vue

```diff
@@ -83,7 +85,7 @@ export default {
   computed: {
```

Comment:
this is older code, but Line 81 : var deletedTableResponses is unused

Reply:
Blame @vietle-cgi :).

Reply:
I can remove this var when I work on MR next sprint